README.md

```diff
@@ -391,6 +391,20 @@ const stringify = fastJson({
 console.log(stringify({ some: 'object' })) // '{"some":"object"}'
 ```
 
+
+<a name="type-coerce"></a>
+## Type coerce
+
+When the object you're serializing does not strictly match the given schema,
+`fast-json-stringify` applies a type coercion as describe below:
+
+- if the schema describes a number and a Infinity, NaN, null or a boolean was given,
```

Comment:
"if the schema describes a number but a .."

Reply:
Done